src/main/java/tr/edu/ogu/ceng/service/SettingService.java

```diff
@@ -24,4 +24,12 @@ public Setting getSetting(String key) throws EntityNotFoundException {
     	}
     	return setting;
     }
+    
+    public String findValueByKey(String key) {
```

Comment:
Bu metodun yukarıdaki getSetting metodundan ne farkı var?

Reply:
Setting objesi döndürmek yerine yalnızca value değerini döndürüyor hocam. Settings içerisinde bulunan created date, updated date gibi özelliklere bu durumda ihtiyaç olmadığı için eklemiştim.